modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/utils/odata.es.js

```diff
@@ -116,27 +134,66 @@ function getConjunctionForGroup(oDataASTNode) {
  * @returns String value of the internal name
  */
 function getExpressionName(oDataASTNode) {
-	return oDataV4ParserNameMap[oDataASTNode.type];
+	const type = oDataASTNode.type;
+
+	let returnValue = oDataV4ParserNameMap[type];
+
+	if (type == 'MethodCallExpression') {
```

Comment:
It might be worth creating a bunch of constants for these string names of the ast nodes. It would help reduce typos